packages/esm-patient-vitals-app/src/biometrics/biometrics-base.scss

```diff
@@ -24,7 +24,7 @@
 .divider {
   width: 1px;
   height: layout.$spacing-05;
-  color: $ui-03;
+  color: $ui-05;
```

Comment:
Ditto

Reply:
When imported as *, it should function properly without requiring the `vars` prefix.